github/tests/Repository.py

```diff
@@ -711,6 +712,13 @@ def testStatistics(self):
     def testGetLicense(self):
         self.assertEqual(len(self.repo.get_license().content), 47646)
 
+    def testGetTopics(self):
+        topic_list = self.repo.get_topics()
+        topic = u'github'
+        self.assertEquals(topic in topic_list, True)
+
+    def testReplaceTopics(self):
+        self.repo.replace_topics(['github', 'testing'])
```

Comment:
Looking at some of the other tests, I think no assertion is needed here? Let me know if this needs to be improved.

Reply:
It's fine